tests/test_workspace_listing_performance.py

```
@@ -0,0 +1,138 @@
+                if conn is not None:
+                    original_execute = conn.execute
+
+                    def _spying_execute(query, params=()):
+                        executed_queries.append(query)
+                        return original_execute(query, params)
+
+                    conn.execute = _spying_execute  # type: ignore[method-assign]
+                yield conn, path
+
+        with patch.object(_ws_db_mod, "open_global_db", _spying_open_global_db):
+            projects, warnings = list_workspace_projects(ws_path, rules=[])
```

Ruff (0.15.15)

[warning] Unpacked variable `projects` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)